configs/components/oifs/oifs.yaml

```diff
@@ -387,69 +498,87 @@ choose_resolution:
                 oasis_grid_name: "048"
                 res_number: 95
                 res_number_tl: "95_3"
+                truncation: "TQ"
```

Comment:
Is OpenIFS capable of running TQ grids? more a question for @joakimkjellsson

Reply:
Should work! 
They are listed at
https://confluence.ecmwf.int/display/OIFS/4.3+OpenIFS%3A+Horizontal+Resolution+and+Configurations
but I don't know anyone using them. 
Not sure why we have it listed in ESM-Tools. I suggest to remove it.

Reply:
Agreed